new_dawn_server/urls.py

```diff
@@ -43,4 +49,4 @@
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
     url(r'^api/', include(v1_api.urls)),
-]
+] + static(settings.STATIC_URL, document_root=settings.STATIC_ROOT) + static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
What does this line do?

Reply:
Good question. This allows users to access static files in those directories via url:
<img width="1434" alt="screen shot 2018-12-20 at 10 44 51 pm" src="https://user-images.githubusercontent.com/8229754/50323721-f13f8700-04a8-11e9-8f2b-28afc50756c4.png">

Reply:
Updated the PR summary to reflect this